setup_f5_gke.sh

```diff
@@ -415,27 +415,30 @@ if [ "$cluster_status" != "0" ] && [ "${PURGE}" == "0" ] && [ "${UPGRADE}" == "0
       SOLR_REPLICAS=3
     fi
 
-    # make sure the compute/zone is updated
-    current_value=$(gcloud config get-value compute/zone)
+    # make sure the compute/region is updated
+    current_value=$(gcloud config get-value compute/region)
     if [ "${current_value}" != "${GCLOUD_REGION}" ]; then
-      gcloud config set compute/zone "${GCLOUD_REGION}"
+      gcloud config set compute/region "${GCLOUD_REGION}"
     fi
 
     gcloud beta container --project "${GCLOUD_PROJECT}" clusters create "${CLUSTER_NAME}" --region "${GCLOUD_REGION}" \
       --no-enable-basic-auth \
       --cluster-version ${GKE_MASTER_VERSION} \
       --machine-type ${INSTANCE_TYPE} \
       --image-type "COS" \
-      --disk-type "pd-standard" --disk-size "100" \
+      --disk-type "pd-standard" \
+      --disk-size "100" \
       --metadata disable-legacy-endpoints=true \
       --scopes "https://www.googleapis.com/auth/devstorage.full_control","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" \
       --num-nodes "1" \
-      --enable-stackdriver-kubernetes \
+      --monitoring=SYSTEM \
```

Comment:
According to https://cloud.google.com/stackdriver/docs/solutions/gke/installing#gcloud_1
`--enable-stackdriver-kubernetes` means `--logging=SYSTEM,WORKLOAD --monitoring=SYSTEM`

According to https://cloud.google.com/stackdriver/docs/solutions/gke/installing#available-logs
default values for `monitoring` is `SYSTEM`, and `SYSTEM,WORKLOAD` fo `logging`. But you explicitly set only for `monitoring`.

Therefore it would great to define either both arguments or none.